packages/table/src/Cell/Cell.types.ts

```diff
@@ -5,40 +5,63 @@ export type Align = Extract<
   'left' | 'right' | 'center'
 >;
 
-interface AlignProp {
+export const CellOverflowBehavior = {
+  Default: 'default',
+  Truncate: 'truncate',
+  // TODO: `Expand`: The cell will expand to the height of its content
+  // Expand: 'expand',
+} as const;
+export type CellOverflowBehavior =
+  (typeof CellOverflowBehavior)[keyof typeof CellOverflowBehavior];
+
+interface BaseCellProps extends HTMLElementProps<'td'> {
   /**
    * Alignment of the cell's contents
    *
    * Overrides `<td>`'s deprecated `align` prop
    */
   align?: Align;
+
+  /**
+   * Defines how a cell should behave when its content is larger than the standard cell height.
+   *
+   * `Default`: The cell height will be fixed to the standard cell height (40px by default).
+   * Any overflowing content will be clipped.
+   *
+   * `Truncate`: The cell height will be fixed to the standard cell height (40px by default),
+   * and include an ellipsis before the content is clipped.
+   *
+   * Note: It's recommended to provide the same value for all cells in a given row.
+   *
+   * @default CellOverflowBehavior.Default
+   */
+  overflow?: CellOverflowBehavior;
```

Comment:
nit: `overflowBehavior` ? overflow makes me think this will be a boolean

Reply:
I had that initially, but wanted to match the similarly-functional prop in Combobox 

https://github.com/mongodb/leafygreen-ui/blob/a19b08de770fe0e4e61aa3a09ea59bae4818c3c1/packages/combobox/src/Combobox.types.ts#L127